var/spack/repos/builtin/packages/nalu/package.py

```diff
@@ -15,9 +15,14 @@ class Nalu(CMakePackage):
     """
 
     homepage = "https://github.com/NaluCFD/Nalu"
+    url = "https://github.com/NaluCFD/Nalu/archive/refs/tags/v1.6.0.tar.gz"
     git = "https://github.com/NaluCFD/Nalu.git"
 
     version("master", branch="master")
+    version("1.6.0", sha256="2eafafe25ed44a7bc1429881f8f944b9794ca51b1e1b29c28a45b91520c7cf97")
```

Comment:
@lucbv  if you still want to have the option to build against `trilinos@develop` you could do something like:
``` python
verison("master", branch="master", preferred=True)
version("develop", branch="master")
depends_on("trilinos@develop", when="@=develop")
```

Reply:
Yes, I saw the `preferred` argument but I'm okay with not adding develop, that's probably better that way as I believe Nalu runs its CI against Trilinos master but not develop. If you all are fine with this I think we can go ahead and merge